src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+
+*** Test cases ***
+
+Test Columns-Titles and Resize
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    ${data-column-size_before2}=  Get Element Attribute  xpath=//*[@id="content"]/div[2]/div/div@data-column-size
+    Click Element  css=i.resizer
+    Wait until element is visible  id=ui-id-1
+    ${data-column-size_before}=  Get Text  xpath=//*[@id="column-size-resize"]/span
+    Drag And Drop By Offset  css=a.ui-slider-handle.ui-state-default.ui-corner-all  -30  0
+    Click Element  css=.ui-dialog-titlebar-close
+    Save Cover Layout
+    Click Link  link=Compose
+    Open Layout Tab
+    ${data-column-size_after2}=  Get Element Attribute  xpath=//*[@id="content"]/div[2]/div/div@data-column-size
+    Click Element  css=i.resizer
+    Wait until element is visible  id=ui-id-1
+    ${data-column-size_after}=  Get Text  xpath=//*[@id="column-size-resize"]/span   
```

Comment:
Right after `Drag And Drop By Offset` you should get the value of this variable. If you close the window, when it opens again, it will have the right value, invalidating the test. The comparison can also be done right after `Drag And Drop By Offset`.

Give this variable a better name.

Reply:
ready @idgserpro , but I repeat, just for giving good feedback, that I do this to close, save and open, because the problems I solved previously had different behaviors after saving.